misc/master.sql

```diff
@@ -512,6 +520,9 @@ ALTER TABLE `shards`
 ALTER TABLE `storageAccounts`
   ADD CONSTRAINT `storageAccounts_ibfk_1` FOREIGN KEY (`userID`) REFERENCES `users` (`userID`) ON DELETE CASCADE ON UPDATE CASCADE;
 
+ALTER TABLE `storageFileLibraries`
+  ADD CONSTRAINT `storageFileLibraries_ibfk_1` FOREIGN KEY (`libraryID`) REFERENCES `libraries` (`libraryID`) ON DELETE CASCADE;
```

Comment:
Oh, we're missing a FK for `storageFileID` as the first constraint. And it seems like that one shouldn't cascade, because we'll only want to delete from `storageFiles` when there are no corresponding rows in `storageFileLibraries`.